wiki/markdown/usage/service.md

```diff
@@ -147,6 +147,9 @@ then in response body we have only one response for that invalid JSON event, it
 
 There is some examples of passing an array of events and what their responses can be
 
+NOTE: Here, introduction of new REMReM property for the size of number of templates in the array passed to generate endpoint. 
+Here, Maximum size of events in array is fetched from REMReM property and checked during generating.
```

Comment:
I think the sentence can be removed, it's implementation detail.

Reply:
Done